packages/dev/core/src/DeviceInput/webDeviceInputSystem.ts

```diff
@@ -765,11 +775,19 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
             }
         };
 
+        // Workaround for MacOS Chromium Browsers for lost pointer capture bug
+        this._pointerMacOSChromeOutEvent = (evt) => {
+            if (this._usingMacOS && !this._usingSafari && evt.buttons > 1) {

```

Comment:
firefox is also available for mac. would be better to filter if it is mac and chromium (and not mac and not safari). Will also be safer for other browsers.

Reply:
Okay, I modified it to check to be more precise